sleap_roots/points.py

```diff
@@ -403,6 +403,37 @@ def filter_plants_with_unexpected_ct(
     return primary_pts, lateral_pts
 
 
+def filter_primary_roots_with_unexpected_count(
+    primary_pts: np.ndarray, expected_count: float
+) -> np.ndarray:
+    """Filter out primary roots if the number of detected roots doesn't match the expected count.
```

Comment:
fix formatting here: description should be a one-liner, indentation should be consistent